From 2f46e97e2c28954c518dd3d5ea8fa0b929e3a05e Mon Sep 17 00:00:00 2001
From: Jannick <thirdedition@gmx.net>
Date: Wed, 25 Nov 2020 17:52:52 +0100
Subject: [PATCH 1/2] igraph-0.8.4: propagate XML2_CFLAGS to Makefile

* src/Makefile.am: define AM_CPPFLAGS to XML2_CFALGS derived by configure.ac
---
 src/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Makefile.am b/src/Makefile.am
index 1e491d472..b04c79765 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -7,6 +7,8 @@ BUILT_SOURCES = foreign-ncol-parser.h foreign-lgl-parser.h \
 # Unfortunately this is not the default behaviour in MinGW/MSYS
 AM_YFLAGS = -d
 
+AM_CPPFLAGS = $(XML2_CFLAGS)
+
 lib_LTLIBRARIES         = libigraph.la
 
 include lapack/blas.inc
-- 
2.29.2.windows.2

